packages/vue-language-core/src/utils/ts.ts

```diff
@@ -158,7 +158,7 @@ function getPartialVueCompilerOptions(
 	}
 	if (rawOptions.plugins) {
 		const plugins = rawOptions.plugins
-			.map<VueLanguagePlugin | undefined>((pluginPath: string) => {
+			.map<VueLanguagePlugin[] | VueLanguagePlugin | undefined>((pluginPath: string) => {
```

Comment:
Supporting deep nested plugins will be even better. `[a, [b, [c, d], e]]`
https://github.com/sxzz/vite/blob/fba97c8119701de256efef581768f98c2348b322/packages/vite/src/node/config.ts#L128-L134

Reply:
Done